### PR TITLE
Support for additional initializers from keras

### PR DESCRIPTION
**Pull request by naveenjafer**
Includes Lecun_Unfiorm, He_Uniform and Glorot_Normal initializers - Reference - https://keras.io/initializers/ 

Completes - 
TODO: Harmonize naming with Keras, and fill in missing entries https://keras.io/initializers/ We should also have He normal/uniform and probably lecun normal/uniform.

Will add Lecun and He Normal once Glorot normal is approved

### CODE REVIEW

File: thinc/initializers.py [hunk 1]
```diff
@@ -29,7 +54,6 @@ def configure_glorot_uniform_init() -> Callable[[Shape], FloatsXd]:
 def zero_init(ops: Ops, shape: Shape) -> FloatsXd:
     return ops.alloc(shape)
 
-
```
Comment (svlandeg) [inline thinc/initializers.py]: As an FYI, we use the `black` formatting tool in Thinc and in spaCy. I believe in this case it would keep these additional lines: it has 2 empty lines between functions declared at the top level. Could you install `black` in your IDE to make sure we're not introducing unnecessary formatting changes in PR's ? cf https://black.readthedocs.io/en/stable/

**Comment by honnibal** [comment]
Thanks a lot for these PRs that fill out these extra pieces of the library!

I want to emphasise that we really appreciate the help. We're trying to keep the library very consistent, so some of our comments might end up nitpicky and the process of merging the PRs might get a little tedious sometimes. Sometimes it might be easier for us to merge first and then make some changes. I hope you won't find this discouraging --- we just want to make sure we keep maintaining consistency.

**Comment by naveenjafer** [comment]
> Thanks a lot for these PRs that fill out these extra pieces of the library!
> 
> I want to emphasise that we really appreciate the help. We're trying to keep the library very consistent, so some of our comments might end up nitpicky and the process of merging the PRs might get a little tedious sometimes. Sometimes it might be easier for us to merge first and then make some changes. I hope you won't find this discouraging --- we just want to make sure we keep maintaining consistency.

Absolutely, I understand :). The least one can do is be patient when using and contributing to an open source repo.

**Comment by naveenjafer** [comment]
> I think this looks great! You could add the few missing ones in this PR, that keeps it nicely together :-)

Thank you for the update about black :) . I have added the remaining initializers and black checked the syntax.

**Comment by svlandeg** [comment]
Looks good! I think this can be merged if Matt agrees :-)
